config/packages/yadm.yaml

```diff
@@ -0,0 +1,2 @@
+yadm:
+    mongo_uri: "%env(PAYUM_MONGO_URI)%"
```

Comment:
configure payment, token storages here. extend base storage class

Reply:
Got it.